### PR TITLE
Add instructions for downloading the nextflow-*-all variant.

### PR DESCRIPTION
**Pull request by abhi18av**
This PR adds instructions in the docs to download the `nextflow-*-variant` 
https://github.com/nextflow-io/nextflow/issues/1602

Here's what the change looks like 

image

### CODE REVIEW

File: docs/getstarted.rst [hunk 1]
```diff
@@ -42,9 +42,10 @@ It only needs two easy steps:
 
 .. note::
     To avoid downloading the dependencies, you could also use the ``nextflow-VERSION-all`` variant available from Github for every Nextflow release version.
-   #. Simply go to the Github releases page and unfold the ``Assets`` section for a release.
+   #. Simply go to the `Github releases page <https://github.com/nextflow-io/nextflow/releases>`__ and unfold the ``Assets`` section for a release.
```
Comment (pditommaso) [inline docs/getstarted.rst:45]: Let's remove also the `Simply` 😄

File: docs/getstarted.rst [hunk 2]
```diff
@@ -22,21 +22,28 @@ Windows system is supported through `WSL <https://en.wikipedia.org/wiki/Windows_
 Installation
 ============
 
-`Nextflow` is distributed as a self-contained executable package, which means that it does not require any special installation procedure.
+`Nextflow` is distributed as an executable package, which means that it does not require any special installation procedure.
```
Comment (pditommaso) [inline docs/getstarted.rst:25]: Let's say instead: * Nextflow is distributed as a self-installer package*

**Comment by abhi18av** [comment]
@pditommaso , I've accomodated the change requests :)

**Comment by pditommaso** [comment]
self-installer was a bit broken :) changed to self-installing. tx!

**Comment by abhi18av** [comment]
No worries :)

**Pull request closed without merging**